units/src/fee.rs

```diff
@@ -80,7 +80,7 @@ impl Amount {
     #[must_use]
     pub const fn checked_div_by_fee_rate_floor(self, fee_rate: FeeRate) -> Option<Weight> {
         match self.to_sat().checked_mul(1000) {
-            Some(amount_msats) => match amount_msats.checked_div(fee_rate.to_sat_per_kwu()) {
+            Some(amount_msats) => match amount_msats.checked_div(fee_rate.to_sat_per_kwu_floor()) {
```

Comment:
In 4831689f0181c458dc6acf118414f56fcfe56bd1:

This one should be `_ceil` (somewhat weirdly, since the function name has `_floor` in it, but that's the nature of division).

Reply:
Mad. Thanks for putting in the thought required to review this PR.